configCivcraft.yml

```diff
@@ -5279,28 +5267,21 @@ recipes:
     fuel_consumption_intervall: 5m
     factory: Basic Pylon
     input:
-      poisonouspotatoes:
-        material: POISONOUS_POTATO
-        amount: 10
       enchantmenttable:
         material: ENCHANTMENT_TABLE
         amount: 5
       ironblocks:
         material: IRON_BLOCK
         amount: 32
-      goldenapples:
-        material: GOLDEN_APPLE
+      goldblocks:
+        material: GOLD_BLOCK
         amount: 16
-      compactedglass:
+      glass:
         material: GLASS
-        amount: 8
-        lore:
-         - Compacted Item
+        amount: 512
       netherquartzore:
         material: QUARTZ_ORE
-        amount: 10
```

Comment:
Beacuse compacting items adds a useless step, same with gold apples, it's needless obfuscation that adds nothing to the process but tedium.